pkg/dbc/parser.go

```diff
@@ -293,11 +293,11 @@ func (p *Parser) float() float64 {
 	}
 	tok := p.nextToken()
 	if tok.typ != scanner.Int && tok.typ != scanner.Float {
-		p.fail(p.peekToken().pos, "expected int or float")
+		p.failf(p.peekToken().pos, "expected int or float")
 	}
 	f, err := strconv.ParseFloat(tok.txt, 64)
 	if err != nil {
-		p.fail(tok.pos, "invalid float")
+		p.failf(tok.pos, "invalid float")
```

Comment:
why failf and not fail on all these, there is no sign in the string?

Reply:
Yes, but we don't have a fail method, only `failf`, so using it for both cases.